Yosemite/Yosemite/Stores/AnnouncementsStore.swift

```diff
@@ -0,0 +1,114 @@
+import Networking
+import Storage
+import WordPressKit
+
+/// Protocol for `AnnouncementsRemote` mainly used for mocking.
+///
+public protocol AnnouncementsRemoteProtocol {
+
+    func getAnnouncements(appId: String,
+                          appVersion: String,
+                          locale: String,
+                          completion: @escaping (Result<[Announcement], Error>) -> Void)
```

Comment:
I will probably stop using `Announcement` and `Feature` from WordPressKit in protocols, etc.
Might create a WooCommerceFeature or something like this to stop spreading the `import WordPressKit` everywhere!

Reply:
In fact, I think I can even **remove** WordPresskit from **Yosemite** and use a protocol to create this boundary 👍